server/config/config.go

```diff
@@ -1115,3 +1120,36 @@ func (c DashboardConfig) ToTiDBTLSConfig() (*tls.Config, error) {
 	}
 	return nil, nil
 }
+
+// ReplicateModeConfig is the configuration for the replicate policy.
+type ReplicateModeConfig struct {
+	ReplicateMode string                    `toml:"replicate-mode" json:"replicate-mode"` // can be 'dr-autosync' or 'majority', default value is 'majority'
+	DRAutoSync    DRAutoSyncReplicateConfig `toml:"dr-autosync" json:"dr-autosync"`       // used when ReplicateMode is 'dr-autosync'
+}
+
+func (c *ReplicateModeConfig) adjust(meta *configMetaData) {
+	if !meta.IsDefined("replicate-mode") {
+		c.ReplicateMode = "majority"
+	}
+	c.DRAutoSync.adjust(meta.Child("dr-autosync"))
+}
+
+// DRAutoSyncReplicateConfig is the configuration for auto sync mode between 2 data centers.
+type DRAutoSyncReplicateConfig struct {
+	LabelKey         string            `toml:"label-key" json:"label-key"`
+	Primary          string            `toml:"primary" json:"primary"`
+	DR               string            `toml:"dr" json:"dr"`
+	MasterReplicas   int               `toml:"primary-replicas" json:"primary-replicas"`
+	DRReplicas       int               `toml:"dr-replicas" json:"dr-replicas"`
+	WaitStoreTimeout typeutil.Duration `toml:"wait-store-timeout" json:"wait-store-timeout"`
+	WaitSyncTimeout  typeutil.Duration `toml:"wait-sync-timeout" json:"wait-sync-timeout"`
+}
+
+func (c *DRAutoSyncReplicateConfig) adjust(meta *configMetaData) {
+	if !meta.IsDefined("wait-store-timeout") {
+		c.WaitStoreTimeout = typeutil.Duration{Duration: time.Minute}
```

Comment:
Need to define constants like other functions?

Reply:
ok